client/src/components/Dashboard.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useContext } from "react";
+import {
+  Grid,
+} from "@material-ui/core";
+
+import { MyContext } from "../Provider";
+import API from "../utils/API";
+import { ItemDisplay } from "./ItemDisplay";
+import { AddItem} from './AddItem'
+import {Header} from './Header'
+
+export function Dashboard() {
+  const {
+    setTopics,
+    fetchPosts,
+    setPosts,
+    posts
+  } = useContext(MyContext)
+  
+  useEffect(() => {
+   fetchPosts()
+    API.getTopics().then((result) => {
+      setTopics(result.data);
+    })
+  }, []);
+
+  return (
+    <div className="dashboard">
+      <Grid container align="center" justify='center' spacing={5 } className={'top-grid'}>
+      <Header/>
+        <ItemDisplay items={posts}/>
+       <AddItem posts={posts} setPosts={setPosts}/>
```

Comment:
Fix the formatting here. `ItemDisplay` and `AddItem` should be on the same vertical line.

Reply:
Also, it's best to add a space before the closing tag of a self-closing element. 
So like this, `<AddItem posts={posts} setPosts={setPosts} />`